xla/service/gpu/transforms/cudnn_custom_call_compiler.cc

```diff
@@ -345,6 +352,23 @@ absl::StatusOr<se::gpu::CudnnGraph> BuildGraphForCustomCallToBackwardFMHA(
                       GetDNNFmhaMaskKindFromCudnnFmhaMaskKind(cudnn_mask_type));
 
   const int sliding_window_length = config.sliding_window_length();
+  auto computations = custom_call->called_computations();
+  const HloComputation *score_mod_fwd_comp = nullptr;
+  const HloComputation *score_mod_bwd_comp = nullptr;
+  if (computations.size() == 1) {
+    score_mod_bwd_comp = computations[0];
+    auto softmax_aux = custom_call->mutable_operand(3);
+    HloInstruction *fwd_custom_call;
+    if (Match(softmax_aux,
+              m::GetTupleElement(m::CustomCall(&fwd_custom_call), 1))) {
+      score_mod_fwd_comp = fwd_custom_call->called_computations()[0];
```

Comment:
Somewhat easier to read due to less nesting:

if (!Match(...)) {
  return absl::InternalError(...)
}
score_mod_fwd_comp = fwd_custom_call->called_computations()[0];

Also, can we be sure that fwd_custom_call actually has exactly 1 computation?